client/app/queue/SubmitDecisionView.jsx

```
@@ -79,13 +95,44 @@ class SubmitDecisionView extends React.PureComponent {
     } = this.props.decision;
     const requiredParams = ['documentId', 'judge'];
 
-    if (decisionType === 'omo') {
-      requiredParams.push('omoType');
+    if (decisionType.includes('OMO')) {
```

I don't love when core business logic relies on includes. Seems brittle to me. For these kind of string comparisons I think it might be better to move them out into constants. Maybe:
```
const OMORequest = 'OMORequest';
...
if (decisiontType === OMORequest) {
...
```

Now you get some level of spell check. If you make a typo, we'll get an error rather than a silent failure.